src/Microsoft.PowerShell.Commands.Diagnostics/resources/GetEventResources.resx

```diff
@@ -253,7 +253,7 @@ The defined template is following:
     <value>The following value is not in a valid DateTime format: {0}.</value>
   </data>
   <data name="LogInfoUnavailable" xml:space="preserve">
```

Comment:
The problem is that this resource is used for any generic Exception that is caught, not just Access Denied.  Perhaps the fix is to add a new resource string and use it specifically for that Exception?

Reply:
There are three uses and all have the same pattern. So new pattern will be good for all the three.